tools/unit-tests-merge.cs

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env /Library/Frameworks/Mono.framework/Commands/csharp
+using System.IO;
+using System.Text;
+using System.Xml;
+
+var args = Environment.GetCommandLineArgs ();
+// first arg is "/Library/Frameworks/Mono.framework/Versions/4.8.0/lib/mono/4.5/csharp.exe"
+// second arg the script itself
+// then comes the ones we care about, the path with all the xunit and nunit files
+
+if (args.Length < 3) {
+    Console.WriteLine ($"Missing path for the direcotry that contains the test results files.");
```

Comment:
```suggestion
    Console.WriteLine ($"Missing path for the directory that contains the test results files.");
```

Reply:
Me and my dyslexia (had to autocorrect it here :P )